primes.rb

```diff
@@ -1,6 +1,21 @@
 class Primes
-  def self.sum_to(limit = 100)
+  def self.sum_to(limit=100)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
+    
+    require 'prime'
+    primes_array = []
+    (0..limit).each do |n|
+        if Prime.prime?(n)
+            primes_array << n
+        end
+    end
+
+    prime_sum = primes_array.inject(:+)
```

Comment:
One reason why this might be slow is that we're having to store all of the prime numbers in an array, and then iterate over it to get our sum. Could we combine these two and do without the array?